tests/test_claude_hook_push_notification.py

```
@@ -0,0 +1,264 @@
+
+
+def resolve_cmux_cli() -> str:
+    explicit = os.environ.get("CMUX_CLI_BIN") or os.environ.get("CMUX_CLI")
+    if explicit:
+        if os.path.exists(explicit) and os.access(explicit, os.X_OK):
+            return explicit
+        raise RuntimeError(f"Configured cmux CLI is not executable: {explicit}")
+
+    candidates: list[str] = []
+    candidates.extend(glob.glob(os.path.expanduser("~/Library/Developer/Xcode/DerivedData/*/Build/Products/Debug/cmux")))
+    candidates.extend(glob.glob("/tmp/cmux-*/Build/Products/Debug/cmux"))
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/cmux-*/Build/Products/Debug/cmux"

(S108)